kolibri/plugins/coach/assets/src/views/recent-page/report-list.vue

```diff
@@ -54,8 +54,8 @@
       'content-icon': require('kolibri.coreVue.components.contentIcon'),
       'progress-bar': require('kolibri.coreVue.components.progressBar'),
     },
-    computed: {
-      reportList() {
+    watch: {
```

Comment:
No, that's not safe - you should never modify the underlying vuex state from a vue file. Also, best to avoid using `watch` - computed properties are more declarative.

We can use [lodash's `orderBy`](https://lodash.com/docs/4.17.4#orderBy):

```javascript
    computed: {
      orderedReports() {
        return _.orderBy(this.reports, 'last_active');
      },
    },
```